pages/nfts/[address].tsx

```diff
@@ -1,8 +1,50 @@
 import React, { useEffect } from 'react';
 import { GetServerSideProps } from 'next';
-import { useNftPageLazyQuery } from '../../src/graphql/indexerTypes';
+import { useNftPageLazyQuery, useWalletProfileLazyQuery } from '../../src/graphql/indexerTypes';
+import { always, equals, ifElse, pipe, length, find, prop } from 'ramda';
+import BN from 'bn.js';
+import cx from 'classnames';
+import { showFirstAndLastFour } from '../../src/modules/utils/string';
+import { useTwitterHandle } from '../../src/common/hooks/useTwitterHandle';
+import Link from 'next/link';
+import Custom404 from '../404';
+import { getPFPFromPublicKey } from '../../src/modules/utils/image';
+import Image from 'next/image';
+import Accordion from '../../src/common/components/elements/Accordion';
 // import Bugsnag from '@bugsnag/js';
 
+const Avatar = ({ address }: { address: string }) => {
+  const { data: twitterHandle } = useTwitterHandle(null, address);
+  console.log('twitterHandle', twitterHandle);
+  const [queryWalletProfile, { data }] = useWalletProfileLazyQuery();
+
+  useEffect(() => {
+    if (!twitterHandle) return;
```

Comment:
What is going on here? It should probably be the other way around, trying graphQL first, and then useTwitterHandle later if graphql did not find it

Reply:
Not sure how we get twitterHandles off of graphql? Is there one supported?  This is the way its done in profiles so I assume it was default

<img width="301" alt="image" src="https://user-images.githubusercontent.com/5829885/160510267-ecded101-3a7d-4944-9a92-7026ab83bcb1.png">